lamindb/_annotate.py

```diff
@@ -410,6 +420,199 @@ def save_artifact(self, description: str, **kwargs) -> Artifact:
         return self._artifact
 
 
+class MuDataAnnotator:
+    """Annotation flow for a ``MuData`` object.
+
+    Args:
+        mdata: The MuData object to annotate.
+        var_index: The registry field for mapping the ``.var`` index for each modality.
+            For example:
+            ``{"modality_1": bt.Gene.ensembl_gene_id, "modality_2": ln.CellMarker.name}``
+        categoricals: A dictionary mapping ``.obs.columns`` to a registry field.
+            For example:
+            ``{"cell_type_ontology_id": bt.CellType.ontology_id, "donor_id": ln.ULabel.name}``
+        using: A reference LaminDB instance.
+    """
+
+    def __init__(
+        self,
+        mdata: MuData,
```

Comment:
I fear this might make it dependent on mudata. Maybe I'm wrong. But you should try in an environment that doesn't have mudata installed.

Reply:
(I know you have if TYPE_CHECKING up there, but still!)